crates/biome_service/src/configuration/parse/json/javascript/formatter.rs

```diff
@@ -0,0 +1,101 @@
+use crate::configuration::javascript::JavascriptFormatter;
+use crate::configuration::PlainIndentStyle;
+use biome_deserialize::json::VisitJsonNode;
+use biome_deserialize::{DeserializationDiagnostic, VisitNode};
+use biome_formatter::LineWidth;
+use biome_js_formatter::context::trailing_comma::TrailingComma;
+use biome_js_formatter::context::{ArrowParentheses, QuoteProperties, QuoteStyle, Semicolons};
+use biome_json_syntax::{JsonLanguage, JsonSyntaxNode};
+use biome_rowan::{AstNode, SyntaxNode};
+
+impl VisitJsonNode for JavascriptFormatter {}
```

Comment:
I was wondering why we need to write it?
Is it for the case when we need to implement its own implementation?
Because we can try to write something like:

`impl<U: VisitNode<JsonLanguage>> VisitJsonNode for U {}`

Reply:
You're the man! Thank you, it will save lot of stuff